CMake/macros.cmake

```diff
@@ -141,6 +141,26 @@ FUNCTION( RUN_QMC_APP TESTNAME SRC_DIR PROCS THREADS TEST_ADDED TEST_LABELS ${AR
 ENDFUNCTION()
 
 
+# this disables the extremely slow setup of full system run tests
+# these prevent a reasonable compile debug loop if CMakeLists.txt files are touched since they
+# far exceed even the feature and compiler checks for time consumed when rerunning cmake.
+#
+# On leconte with rerun of cmake with
+# slow tests 27 secs to run cmake
+# with them disable 8 secs
+
+SET( QMC_DISABLE_SLOW_TESTS FALSE CACHE BOOL "Disable the slow cmake system test additions")
+
+IF (QMC_DISABLE_SLOW_TESTS)
+  FUNCTION(QMC_RUN_AND_CHECK BASE_NAME BASE_DIR PREFIX INPUT_FILE PROCS THREADS SHOULD_SUCCEED)
```

Comment:
I think the arguments can be removed from these empty functions.  That avoids inconsistencies in the argument lists if the functions are changed in the future (or avoids the need to update two places).
CMake requires functions to be called with the same number or more arguments, so the calls will work fine.

Reply:
Nice trick, done.